opensearch_dashboards.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "indexManagementDashboards",
-  "version": "2.4.0.0",
-  "opensearchDashboardsVersion": "2.4",
+  "version": "2.4.0",
```

Comment:
Not sure why is this change. Plugin uses 4 digits as its version number

Reply:
This comes from using "plugin-helpers" API in OpenSearch Dashboards. For example: `yarn plugin-helpers version --sync` will update your plugin's package.json and opensearch_dashboards.json to match version of OSD

Reply:
I see, let's not change this.